specs/glossary.md

```diff
@@ -1,3 +1,4 @@
+<!-- markdownlint-disable MD053 -->
```

Comment:
Why is MD053 disabled in this file?

Reply:
MD053 is a rule that `Link and image reference definitions should be needed`.
Without it, we need to remove all unused definitions such as `[storage-root]: glossary.md#storage-root` in this glossary file. But I think it is kind of convention to add the definition of all glossaries to use in future. So I suppressed this rule only for this glossary file.